content/hardware/02.hero/boards/uno-r4-wifi/tutorials/touch/content.md

```diff
@@ -0,0 +1,130 @@
+| A2 (D16)     | 22                         | 2                                | (1 << 6)                           |
+| LOVE_BUTTON  | 0                          | 0                                | (1 << 0)                           |
+
+**Arduino® UNO-R4 WiFi:**
+| Arduino Pin  | Touch Sensor Channel (TS#) | Channel Control Index (CHAC idx) | Channel Control Bit Mask (CHAC val) |
+|--------------|----------------------------|----------------------------------|-------------------------------------|
+| D0           | 9                          | 1                                | (1 << 1)                           |
+| D1           | 8                          | 1                                | (1 << 0)                           |
+| D2           | 13                         | 1                                | (1 << 5)                           |
+| D3           | 34                         | 4                                | (1 << 2)                           |
+| D6           | 12                         | 1                                | (1 << 4)                           |
+| D8           | 11                         | 1                                | (1 << 3)                           |
+| D9           | 2                          | 0                                | (1 << 2)                           |
+| D11          | 7                          | 0                                | (1 << 7)                           |
+| D12          | 6                          | 0                                | (1 << 6)                           |
+| A1 (D15)     | 21                         | 2                                | (1 << 5)                           |
+| A2 (D16)     | 22                         | 2                                | (1 << 6)                           |
+| LOVE_BUTTON  | 27                         | 3                                | (1 << 3)                           |
+
+
+### Library Functions
+
+The Arduino_CapacitiveTouch library provides several functions to work with capacitive touch inputs:
+
+- **```CapacitiveTouch(uint8_t pin)```** Constructs a capacitive touch sensor for the given pin.
+- **```bool begin()```** Initializes the sensor and configures the pin and hardware.
+- **```int read()```** Reads the raw sensor value and returns it.
+- **```bool isTouched()```** Checks if the sensor is touched based on the threshold.
+- **```void setThreshold(int threshold)```** Sets the detection threshold for touch sensitivity.
+- **```int getThreshold()```** Retrieves the current detection threshold.
+
+For more detailed usage and examples, refer to the [CapacitiveSensor library documentation](https://docs.arduino.cc/libraries/capacitivesensor/).
+
+## Example Code
+
+Here's a simple example to get you started with capacitive sensing on the Uno R4.
+For this example we are connecting a single piece of any conductive material to the pin ```D0``` on the Board.
+
+![How to connect](assets/HoockupGuideExample.png)
```

Comment:
I think it would be nice to move this to a separate and dedicated section, so the readers can assemble the hardware and understand how it works electrically. Then they can move to example section to upload the code and test it.